### PR TITLE
Md3672 Amazon Sales Channel

### PR DESCRIPTION
**Pull request by lorikrell**
## Purpose of this PR

This PR adds installation and release notes for the Amazon Sales Channel PreRelease.

## Affected URLs

- https://devdocs.magento.com/extensions/amazon-sales/
- https://devdocs.magento.com/extensions/amazon-sales/release-notes

whatsnew
Added Amazon Sales Channel [installation guide](https://devdocs.magento.com/extensions/amazon-sales/) and [release notes](https://devdocs.magento.com/extensions/amazon-sales/release-notes).

### CODE REVIEW

File: extensions/amazon-sales/release-notes/index.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+group: extensions
+title: Amazon Sales Channel Release Notes
+---
+
+*Release notes published April 2019.*
```
Comment (jeff-matthews) [inline extensions/amazon-sales/release-notes/index.md]: You don't really need this anymore now that we add the last modified date to each topic. It's up to you though.

File: extensions/amazon-sales/release-notes/index.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+**Amazon Sales Channel** is now available with Magento 2.2.x and 2.3.x. Merchants can use Amazon Sales Channel to integrate with an Amazon Seller account to post and sell Magento products on Amazon.
```
Comment (jeff-matthews) [inline extensions/amazon-sales/release-notes/index.md]: "Magento" products? 

Maybe this?

> "...post and sell products in your Magento catalog on Amazon."

Also, is this available for both Open Source and Commerce? If so, use similar language as line 24. If not, be specific and use a jekyll variable for the product name. See line 24.

File: extensions/amazon-sales/release-notes/index.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+- {:.new} **Intelligent Pricing**: Set automated repricing rules to increase your chances for the coveted Buy Box. Set prices to dynamically adjust to the current Buy Box price, or lowest competitor pricing. Set limits to repricing to protect your margin.
+
+- {:.new} **Listing Management**: Automate product listings and sync your Magento catalog to the Amazon Marketplace using listing rules. Add specific overrides to finely control your offerings. Monitor and manage all your listings directly from the Magento Admin.
+
+- {:.new} **Consistent Inventory Management**: Keep your Magento and Amazon inventory quantities in constant synchronization.
+
+- {:.new} **Order and Fulfillment Management**: Track Amazon orders through the dashboard, with seamless communication and inventory updates. Complete and track order shipments as fulfilled by Amazon, merchant fulfilled, or a mix of methods.
+
+-   {:.bug} Updates for product quantity may take ~2 hours to sync.
```
Comment (jeff-matthews) [inline extensions/amazon-sales/release-notes/index.md]: The descriptions make it sound like these are known issues that haven't been fixed. If so, change the description of the bug icon above. If not, I think they should be worded like they've been fixed. However, as an inaugural release, I'm not sure why we would list "bug" fixes.

File: extensions/google-shopping-ads/index.md [hunk 4]
```diff
@@ -7,7 +7,7 @@ The Google Shopping ads extension installs and adds Google Shopping features to
 
 ## Requirements
 
-- **Magento Instance**: Google Shopping Ads Channel can be installed on instances with {{site.data.var.ce}} and {{site.data.var.ee}} versions 2.2.X and 2.3.X. We do not support the extension on Magento 2.1 or Magento 1.
+- **Magento Instance**: Google Shopping ads Channel can be installed on instances with {{site.data.var.ce}} and {{site.data.var.ee}} versions 2.2.X and 2.3.X. We do not support the extension on Magento 2.1 or Magento 1.
```
Comment (jeff-matthews) [inline extensions/google-shopping-ads/index.md]: Is this available for both Open Source and Commerce? If so, use similar language as line 24. If not, be specific and use a jekyll variable for the product name.

File: extensions/amazon-sales/release-notes/index.md (hunk 1 above)
Comment (lorikrell) [reply extensions/amazon-sales/release-notes/index.md]: Gotcha, cutting

File: extensions/amazon-sales/release-notes/index.md (hunk 3 above)
Comment (lorikrell) [reply extensions/amazon-sales/release-notes/index.md]: Understood, this was requested for the prerelease. Will look them over and consider some rewording. Thanks!

**Comment by lorikrell** [comment]
running tests

**Pull request merged**